README.md

```diff
@@ -0,0 +1,102 @@
+# The Bounded Context Canvas
+
+The Bounded Context Canvas is a collaborative tool for designing and documenting the design of a single bounded context.
+
+A [bounded context](https://martinfowler.com/bliki/BoundedContext.html) is a sub-system in a software architecture aligned to a part of your domain.
+
+The canvas guides you through the process of designing a bounded context by requiring you to consider and make choices about the key elements of its design, from naming to responsibilities, to its public interface and dependencies.
+
+![alt text](resources/bounded-context-canvas-v3.jpeg "The Bounded Context Canvas V3")
+
+## How to Use
+
+There is no strict rule about the order in which you should fill out the canvas, however, in general it is recommended to start at the top left with the name, move down the left column and then start at the top of the right column.
+
+You may not have all the information you need to complete certain sections of the canvas. In such a case, you'll need to use other modelling techniques to find the information you require. See [DDD Toolbox](https://github.com/ddd-crew/ddd-toolbox) for suggestions.
+
+## Section Definitions
+
+Here is a short explanation of each section of the canvas.
+
+### Name
+Naming is hard. Writing down the name of your context and gaining agreement as a team will frame how you design the context.
+
+### Description
+A few sentences describing the why and what of the context in business language. No technical details here.
+
+Writing down the description forces you to clearly articulate fuzzy thoughts and ensure everybody in the team is on the same page.
+
+### Strategic Classification
+How important is this context to the success of your organisation: 
+
+- core domain: a key strategic initiative
+- supporting domain: necessary but not a differentiator
+- generic: a common capability found in many domains
+
+What role does the context play in your business model:
+
+- revenue generator: people pay directly for this
+- engagement creator: users like it but they don't pay for it
+- compliance enforcer: protects your business reputation and existence
+
+How evolved is the concept (see [Wardley Maps](https://medium.com/wardleymaps)):
+
+- genesis: new unexplored domain
+- custom built: companies are building their own versions
+- product: off-the-shelf versions exist with differentiation
+- commodity: highly-standardised versions exist 
+
+### Business Decisions
+What are the key business rules and policies within this context?
+
+### Ubiquitous Language
+What are the key domain terms that exist within this context, and what do they mean?
+
+### Model Traits
```

Comment:
We could add to the repo the model trait cheat-sheet we use during our workshop.

Reply:
Pushed.